doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -114,6 +114,16 @@ nRF9160: Asset Tracker v2
 * Updated:
 
   * The application now uses the new LwM2M location assistance objects through the :ref:`lib_lwm2m_location_assistance` library.
+  * The application now uses the :ref:`lib_location` library for retrieving location information.
+  
+    This is a major change in the application code but a minor change from the user perspective.
+    * Added location module and removed GNSS module. GNSS used through Location library.
+    * Neighbor cell handling moved from modem module to location module to be used through Location library
+    * The location module triggers a ``location_request()`` with GNSS being the first priority method and cellular the second priority if they are enabled in the application configuration
+    * NMEA support removed. This was there only because originally, nRF Cloud did not support PVT but that has changed.
+    * AGPS/PGPS are not requested based on triggers in the application but only based on Location library events LOCATION_EVT_GNSS_ASSISTANCE_REQUEST and LOCATION_EVT_GNSS_PREDICTION_REQUEST
+    * Currently, you cannot configure or define the LTE LC neighbor cell search type with the Location library. The default search type is always used.
+    * The Kconfig option ``CONFIG_GNSS_MODULE_PGPS_STORE_LOCATION`` (calling ``nrf_cloud_pgps_set_location()``) is not supported in the Location library.
```

Comment:
Not sure if we should mention this option now that it's removed

Reply:
I thought we are supposed to mention if we remove some functionality. @peknis @simensrostad What to do?

Reply:
Right, I'll defer to @peknis